Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -61,14 +62,33 @@ void ArrayPrototype::initialize(GlobalObject& global_object)
     define_native_function(vm.names.fill, fill, 1, attr);
     define_native_function(vm.names.values, values, 0, attr);
     define_native_function(vm.names.flat, flat, 0, attr);
+    define_native_function(vm.names.flatMap, flat_map, 1, attr);
     define_native_function(vm.names.at, at, 1, attr);
     define_native_function(vm.names.keys, keys, 0, attr);
+    define_native_function(vm.names.entries, entries, 0, attr);
+    define_native_function(vm.names.copyWithin, copy_within, 2, attr);
 
     // Use define_property here instead of define_native_function so that
     // Object.is(Array.prototype[Symbol.iterator], Array.prototype.values)
     // evaluates to true
     // 23.1.3.33 Array.prototype [ @@iterator ] ( ), https://tc39.es/ecma262/#sec-array.prototype-@@iterator
     define_property(vm.well_known_symbol_iterator(), get(vm.names.values), attr);
+
+    // 23.1.3.34 Array.prototype [ @@unscopables ], https://tc39.es/ecma262/#sec-array.prototype-@@unscopables
+    Object* unscopable_list = create_empty(global_object);
+    unscopable_list->set_prototype(nullptr);
+    unscopable_list->put(vm.names.copyWithin, Value(true));
+    unscopable_list->put(vm.names.entries, Value(true));
+    unscopable_list->put(vm.names.fill, Value(true));
+    unscopable_list->put(vm.names.find, Value(true));
+    unscopable_list->put(vm.names.findIndex, Value(true));
+    unscopable_list->put(vm.names.flat, Value(true));
+    unscopable_list->put(vm.names.flatMap, Value(true));
+    unscopable_list->put(vm.names.includes, Value(true));
+    unscopable_list->put(vm.names.keys, Value(true));
+    unscopable_list->put(vm.names.values, Value(true));
+
+    define_property(vm.well_known_symbol_unscopables(), unscopable_list->value_of(), Attribute::Configurable);
```

Comment:
```suggestion
    define_property(vm.well_known_symbol_unscopables(), unscopable_list, Attribute::Configurable);
```

An `Object*` will be wrapped in a `Value` implicitly, no need for `value_of()`.